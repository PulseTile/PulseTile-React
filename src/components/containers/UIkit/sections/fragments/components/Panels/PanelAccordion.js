import React, { Component } from 'react';

const panelClassNameDefault = 'panel panel-secondary';

export default class PanelAccordion extends Component {

    state = {
        panelFirstClassName: panelClassNameDefault,
        panelSecondClassName: panelClassNameDefault,
        openPanel: null,
    };

    openPanel = panel => {
        this.closePanels();
        if ('first' === panel) {
            this.setState({
                panelFirstClassName: panelClassNameDefault + ' open',
                openPanel: 'first',
            })
        }
        if ('second' === panel) {
            this.setState({
                panelSecondClassName: panelClassNameDefault + ' open',
                openPanel: 'second',
            })
        }
    };

    closePanels = () => {
        this.setState({
            panelFirstClassName: panelClassNameDefault,
            panelSecondClassName: panelClassNameDefault,
            openPanel: null,
        })
    };

    render() {
        const { panelFirstClassName, panelSecondClassName } = this.state;
        return (
            <div className="col-xs-12 col-sm-6">
                <div className="ui-sub-section">
                    <div className="ui-sub-title">Panel accordion</div>
                    <div className="panel-group accordion">
                        <div className={panelFirstClassName}>
                            <div className="panel-heading">
                                <div className="control-group right" >
                                    <button type="button" className="btn btn-success btn-inverse btn-square btn-toggle-rotate" onClick={() => this.openPanel('first')}>
                                        <i className="btn-icon fa fa-chevron-up"></i>
                                    </button>
                                </div>
                                <h3 className="panel-title">Panel title 1</h3>
                            </div>
                            <div className="panel-body">
                                <div className="ui-panel-content">Panel Content 1</div>
                                <div className="panel-control">
                                    <div className="wrap-control-group hide-indent-bottom">
                                        <div className="control-group with-indent right">
                                            <button className="btn btn-success btn-inverse btn-create">
                                                <i className="btn-icon fa fa-plus"></i> <span className="btn-text">Create</span>
                                            </button>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div className={panelSecondClassName}>
                            <div className="panel-heading">
                                <div className="control-group right" >
                                    <button className="btn btn-success btn-inverse btn-square btn-toggle-rotate" onClick={() => this.openPanel('second')}>
                                        <i className="btn-icon fa fa-chevron-up"></i>
                                    </button>
                                </div>
                                <h3 className="panel-title">Panel title 2</h3>
                            </div>
                            <div className="panel-body">
                                <div className="ui-panel-content">Panel Content 2</div>
                                <div className="panel-control">
                                    <div className="wrap-control-group hide-indent-bottom">
                                        <div className="control-group with-indent right">
                                            <button className="btn btn-success btn-inverse btn-create">
                                                <i className="btn-icon fa fa-plus"></i> <span className="btn-text">Create</span>
                                            </button>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        );
    }
}