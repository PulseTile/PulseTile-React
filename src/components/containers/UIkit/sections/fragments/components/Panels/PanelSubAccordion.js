import React, { Component } from 'react';

const panelClassNameDefault = 'panel panel-secondary panel-subaccordion accordion';

/**
 * This component return Panel Sub-Accordion
 */
export default class PanelSubAccordion extends Component {

  state = {
    panelFirstClassName: panelClassNameDefault,
    panelSecondClassName: panelClassNameDefault,
    isFirstOpen: false,
    isSecondOpen: false,
  };

  toggleFirstSubAccordion = () => {
    const { isFirstOpen } = this.state;
    if (isFirstOpen) {
      this.setState({
        panelFirstClassName: panelClassNameDefault,
        isFirstOpen: !isFirstOpen
      })
    } else {
      this.setState({
        panelFirstClassName: panelClassNameDefault + ' open',
        isFirstOpen: !isFirstOpen
      })
    }
  };

  toggleSecondSubAccordion = () => {
    const { isSecondOpen } = this.state;
    if (isSecondOpen) {
      this.setState({
        panelSecondClassName: panelClassNameDefault,
        isSecondOpen: !isSecondOpen
      })
    } else {
      this.setState({
        panelSecondClassName: panelClassNameDefault + ' open',
        isSecondOpen: !isSecondOpen
      })
    }
  };

  render() {
    const { panelFirstClassName, panelSecondClassName } = this.state;
    return (
      <div className="col-xs-12 col-sm-6">
        <div className="ui-sub-section">
          <div className="ui-sub-title">Panel sub-accordion</div>
          <div className={panelFirstClassName}>
            <div className="panel-heading" onClick={() => this.toggleFirstSubAccordion()}>
              <h3 className="panel-title">Panel title 1</h3>
            </div>
            <div className="panel-body">
              <div className="ui-panel-content">Panel Content 1</div>
              <div className="panel-control">
                <div className="wrap-control-group hide-indent-bottom">
                  <div className="control-group with-indent right">
                    <button className="btn btn-success btn-inverse btn-create">
                      <i className="btn-icon fa fa-plus"></i> <span className="btn-text">Create</span>
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div className={panelSecondClassName}>
            <div className="panel-heading" onClick={() => this.toggleSecondSubAccordion()}>
              <h3 className="panel-title">Panel title 2</h3>
            </div>
            <div className="panel-body">
              <div className="ui-panel-content">Panel Content 2</div>
              <div className="panel-control">
                <div className="wrap-control-group hide-indent-bottom">
                  <div className="control-group with-indent right">
                    <button className="btn btn-success btn-inverse btn-create">
                     <i className="btn-icon fa fa-plus"></i> <span className="btn-text">Create</span>
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    );
  }
};
