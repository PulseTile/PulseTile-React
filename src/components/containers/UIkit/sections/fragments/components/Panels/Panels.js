import React from 'react';
import PanelAccordion from './PanelAccordion';
import PanelSubAccordion from './PanelSubAccordion';

/**
 * This component returns Panels section for COMPONENT
 */
const Panels = () => {
    return (
        <div id="panels" className="ui-section">
            <strong className="ui-title">Panels</strong>
            <div className="row">
                <div className="col-xs-12 col-sm-6">
                    <div className="ui-sub-section">
                        <div className="ui-sub-title">Panel primary</div>
                        <div className="panel panel-primary">
                            <div className="panel-heading">
                                <div className="control-group right" >
                                    <button className="btn btn-success btn-inverse btn-square hidden-xs hidden-sm btn-expand-panel"><i className="btn-icon fa fa-expand"></i><i className="btn-icon fa fa-compress"></i></button>
                                </div>
                                <h3 className="panel-title">Panel title</h3>
                            </div>
                            <div className="panel-body">
                                <div className="ui-panel-content">Panel Conten</div>
                                <div className="panel-control">
                                    <div className="wrap-control-group hide-indent-bottom">
                                        <div className="control-group with-indent right">
                                            <button className="btn btn-success btn-inverse btn-create"><i className="btn-icon fa fa-plus"></i> <span className="btn-text">Create</span></button>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <div className="col-xs-12 col-sm-6">
                    <div className="ui-sub-section">
                        <div className="ui-sub-title">Panel secondary</div>
                        <div className="panel panel-secondary">
                            <div className="panel-heading">
                                <div className="control-group right" >
                                    <button className="btn btn-success btn-inverse btn-square hidden-xs hidden-sm btn-expand-panel"><i className="btn-icon fa fa-expand"></i><i className="btn-icon fa fa-compress"></i></button>
                                </div>
                                <h3 className="panel-title">Panel title</h3>
                            </div>
                            <div className="panel-body">
                                <div className="ui-panel-content">Panel Conten</div>
                                <div className="panel-control">
                                    <div className="wrap-control-group hide-indent-bottom">
                                        <div className="control-group with-indent right">
                                            <button className="btn btn-success btn-inverse btn-create"><i className="btn-icon fa fa-plus"></i> <span className="btn-text">Create</span></button>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <div className="row">
                <div className="col-xs-12 col-sm-6">
                    <div className="ui-sub-section">
                        <div className="ui-sub-title">Panel secondary</div>
                        <div className="panel panel-secondary">
                            <div className="panel-heading">
                                <div className="control-group right" >
                                    <button className="btn btn-success btn-inverse btn-square hidden-xs hidden-sm btn-expand-panel"><i className="btn-icon fa fa-expand"></i><i className="btn-icon fa fa-compress"></i></button>
                                </div>
                                <h3 className="panel-title">Panel title</h3>
                            </div>
                            <div className="panel-body">
                                <div className="panel-body-inner">
                                    <div className="ui-panel-content">Panel Conten</div>
                                </div>
                                <div className="panel-control">
                                    <div className="wrap-control-group hide-indent-bottom">
                                        <div className="control-group with-indent right">
                                            <button className="btn btn-success btn-inverse btn-create"><i className="btn-icon fa fa-plus"></i> <span className="btn-text">Create</span></button>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <div className="col-xs-12 col-sm-6">
                    <div className="ui-sub-section">
                        <div className="ui-sub-title">Panel small</div>
                        <div className="panel panel-secondary panel-small">
                            <div className="panel-heading">
                                <h3 className="panel-title">Panel title</h3>
                            </div>
                            <div className="panel-body">
                                <div className="ui-panel-content">Panel Conten</div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>

            <div className="row">
                <PanelAccordion />
                <PanelSubAccordion />
            </div>
        </div>
    );
};

export default Panels;
