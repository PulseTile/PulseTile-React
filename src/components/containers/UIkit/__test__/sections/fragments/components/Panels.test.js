import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Panels from '../../../../sections/fragments/components/Panels/Panels';
import PanelAccordion from '../../../../sections/fragments/components/Panels/PanelAccordion';
import PanelSubAccordion from '../../../../sections/fragments/components/Panels/PanelSubAccordion';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <Panels />', () => {
  it('should renders Panels with props correctly', () => {
    const component = shallow(<Panels />);
    expect(component).toMatchSnapshot();
  });
});

describe('Component <PanelAccordion />', () => {
  it('should renders PanelAccordion with props correctly', () => {
    const component = shallow(<PanelAccordion />);
    expect(component).toMatchSnapshot();
    component.find('.btn-toggle-rotate').at(0).simulate('click');
    expect(component).toMatchSnapshot();
    component.find('.btn-toggle-rotate').at(1).simulate('click');
    expect(component).toMatchSnapshot();
  });
});

describe('Component <PanelSubAccordion />', () => {
  it('should renders PanelSubAccordion with props correctly', () => {
    const component = shallow(<PanelSubAccordion />);
    expect(component).toMatchSnapshot();

    component.find('.panel-heading').at(0).simulate('click');
    expect(component).toMatchSnapshot();
    component.find('.panel-heading').at(0).simulate('click');

    component.find('.panel-heading').at(1).simulate('click');
    expect(component).toMatchSnapshot();
    component.find('.panel-heading').at(1).simulate('click');
  });
});
