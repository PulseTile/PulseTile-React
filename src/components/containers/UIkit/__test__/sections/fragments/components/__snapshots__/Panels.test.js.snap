// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <PanelAccordion /> should renders PanelAccordion with props correctly 1`] = `
<div
  className="col-xs-12 col-sm-6"
>
  <div
    className="ui-sub-section"
  >
    <div
      className="ui-sub-title"
    >
      Panel accordion
    </div>
    <div
      className="panel-group accordion"
    >
      <div
        className="panel panel-secondary"
      >
        <div
          className="panel-heading"
        >
          <div
            className="control-group right"
          >
            <button
              className="btn btn-success btn-inverse btn-square btn-toggle-rotate"
              onClick={[Function]}
              type="button"
            >
              <i
                className="btn-icon fa fa-chevron-up"
              />
            </button>
          </div>
          <h3
            className="panel-title"
          >
            Panel title 1
          </h3>
        </div>
        <div
          className="panel-body"
        >
          <div
            className="ui-panel-content"
          >
            Panel Content 1
          </div>
          <div
            className="panel-control"
          >
            <div
              className="wrap-control-group hide-indent-bottom"
            >
              <div
                className="control-group with-indent right"
              >
                <button
                  className="btn btn-success btn-inverse btn-create"
                >
                  <i
                    className="btn-icon fa fa-plus"
                  />
                   
                  <span
                    className="btn-text"
                  >
                    Create
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="panel panel-secondary"
      >
        <div
          className="panel-heading"
        >
          <div
            className="control-group right"
          >
            <button
              className="btn btn-success btn-inverse btn-square btn-toggle-rotate"
              onClick={[Function]}
            >
              <i
                className="btn-icon fa fa-chevron-up"
              />
            </button>
          </div>
          <h3
            className="panel-title"
          >
            Panel title 2
          </h3>
        </div>
        <div
          className="panel-body"
        >
          <div
            className="ui-panel-content"
          >
            Panel Content 2
          </div>
          <div
            className="panel-control"
          >
            <div
              className="wrap-control-group hide-indent-bottom"
            >
              <div
                className="control-group with-indent right"
              >
                <button
                  className="btn btn-success btn-inverse btn-create"
                >
                  <i
                    className="btn-icon fa fa-plus"
                  />
                   
                  <span
                    className="btn-text"
                  >
                    Create
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <PanelAccordion /> should renders PanelAccordion with props correctly 2`] = `
<div
  className="col-xs-12 col-sm-6"
>
  <div
    className="ui-sub-section"
  >
    <div
      className="ui-sub-title"
    >
      Panel accordion
    </div>
    <div
      className="panel-group accordion"
    >
      <div
        className="panel panel-secondary open"
      >
        <div
          className="panel-heading"
        >
          <div
            className="control-group right"
          >
            <button
              className="btn btn-success btn-inverse btn-square btn-toggle-rotate"
              onClick={[Function]}
              type="button"
            >
              <i
                className="btn-icon fa fa-chevron-up"
              />
            </button>
          </div>
          <h3
            className="panel-title"
          >
            Panel title 1
          </h3>
        </div>
        <div
          className="panel-body"
        >
          <div
            className="ui-panel-content"
          >
            Panel Content 1
          </div>
          <div
            className="panel-control"
          >
            <div
              className="wrap-control-group hide-indent-bottom"
            >
              <div
                className="control-group with-indent right"
              >
                <button
                  className="btn btn-success btn-inverse btn-create"
                >
                  <i
                    className="btn-icon fa fa-plus"
                  />
                   
                  <span
                    className="btn-text"
                  >
                    Create
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="panel panel-secondary"
      >
        <div
          className="panel-heading"
        >
          <div
            className="control-group right"
          >
            <button
              className="btn btn-success btn-inverse btn-square btn-toggle-rotate"
              onClick={[Function]}
            >
              <i
                className="btn-icon fa fa-chevron-up"
              />
            </button>
          </div>
          <h3
            className="panel-title"
          >
            Panel title 2
          </h3>
        </div>
        <div
          className="panel-body"
        >
          <div
            className="ui-panel-content"
          >
            Panel Content 2
          </div>
          <div
            className="panel-control"
          >
            <div
              className="wrap-control-group hide-indent-bottom"
            >
              <div
                className="control-group with-indent right"
              >
                <button
                  className="btn btn-success btn-inverse btn-create"
                >
                  <i
                    className="btn-icon fa fa-plus"
                  />
                   
                  <span
                    className="btn-text"
                  >
                    Create
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <PanelAccordion /> should renders PanelAccordion with props correctly 3`] = `
<div
  className="col-xs-12 col-sm-6"
>
  <div
    className="ui-sub-section"
  >
    <div
      className="ui-sub-title"
    >
      Panel accordion
    </div>
    <div
      className="panel-group accordion"
    >
      <div
        className="panel panel-secondary"
      >
        <div
          className="panel-heading"
        >
          <div
            className="control-group right"
          >
            <button
              className="btn btn-success btn-inverse btn-square btn-toggle-rotate"
              onClick={[Function]}
              type="button"
            >
              <i
                className="btn-icon fa fa-chevron-up"
              />
            </button>
          </div>
          <h3
            className="panel-title"
          >
            Panel title 1
          </h3>
        </div>
        <div
          className="panel-body"
        >
          <div
            className="ui-panel-content"
          >
            Panel Content 1
          </div>
          <div
            className="panel-control"
          >
            <div
              className="wrap-control-group hide-indent-bottom"
            >
              <div
                className="control-group with-indent right"
              >
                <button
                  className="btn btn-success btn-inverse btn-create"
                >
                  <i
                    className="btn-icon fa fa-plus"
                  />
                   
                  <span
                    className="btn-text"
                  >
                    Create
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="panel panel-secondary open"
      >
        <div
          className="panel-heading"
        >
          <div
            className="control-group right"
          >
            <button
              className="btn btn-success btn-inverse btn-square btn-toggle-rotate"
              onClick={[Function]}
            >
              <i
                className="btn-icon fa fa-chevron-up"
              />
            </button>
          </div>
          <h3
            className="panel-title"
          >
            Panel title 2
          </h3>
        </div>
        <div
          className="panel-body"
        >
          <div
            className="ui-panel-content"
          >
            Panel Content 2
          </div>
          <div
            className="panel-control"
          >
            <div
              className="wrap-control-group hide-indent-bottom"
            >
              <div
                className="control-group with-indent right"
              >
                <button
                  className="btn btn-success btn-inverse btn-create"
                >
                  <i
                    className="btn-icon fa fa-plus"
                  />
                   
                  <span
                    className="btn-text"
                  >
                    Create
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <PanelSubAccordion /> should renders PanelSubAccordion with props correctly 1`] = `
<div
  className="col-xs-12 col-sm-6"
>
  <div
    className="ui-sub-section"
  >
    <div
      className="ui-sub-title"
    >
      Panel sub-accordion
    </div>
    <div
      className="panel panel-secondary panel-subaccordion accordion"
    >
      <div
        className="panel-heading"
        onClick={[Function]}
      >
        <h3
          className="panel-title"
        >
          Panel title 1
        </h3>
      </div>
      <div
        className="panel-body"
      >
        <div
          className="ui-panel-content"
        >
          Panel Content 1
        </div>
        <div
          className="panel-control"
        >
          <div
            className="wrap-control-group hide-indent-bottom"
          >
            <div
              className="control-group with-indent right"
            >
              <button
                className="btn btn-success btn-inverse btn-create"
              >
                <i
                  className="btn-icon fa fa-plus"
                />
                 
                <span
                  className="btn-text"
                >
                  Create
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="panel panel-secondary panel-subaccordion accordion"
    >
      <div
        className="panel-heading"
        onClick={[Function]}
      >
        <h3
          className="panel-title"
        >
          Panel title 2
        </h3>
      </div>
      <div
        className="panel-body"
      >
        <div
          className="ui-panel-content"
        >
          Panel Content 2
        </div>
        <div
          className="panel-control"
        >
          <div
            className="wrap-control-group hide-indent-bottom"
          >
            <div
              className="control-group with-indent right"
            >
              <button
                className="btn btn-success btn-inverse btn-create"
              >
                <i
                  className="btn-icon fa fa-plus"
                />
                 
                <span
                  className="btn-text"
                >
                  Create
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <PanelSubAccordion /> should renders PanelSubAccordion with props correctly 2`] = `
<div
  className="col-xs-12 col-sm-6"
>
  <div
    className="ui-sub-section"
  >
    <div
      className="ui-sub-title"
    >
      Panel sub-accordion
    </div>
    <div
      className="panel panel-secondary panel-subaccordion accordion open"
    >
      <div
        className="panel-heading"
        onClick={[Function]}
      >
        <h3
          className="panel-title"
        >
          Panel title 1
        </h3>
      </div>
      <div
        className="panel-body"
      >
        <div
          className="ui-panel-content"
        >
          Panel Content 1
        </div>
        <div
          className="panel-control"
        >
          <div
            className="wrap-control-group hide-indent-bottom"
          >
            <div
              className="control-group with-indent right"
            >
              <button
                className="btn btn-success btn-inverse btn-create"
              >
                <i
                  className="btn-icon fa fa-plus"
                />
                 
                <span
                  className="btn-text"
                >
                  Create
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="panel panel-secondary panel-subaccordion accordion"
    >
      <div
        className="panel-heading"
        onClick={[Function]}
      >
        <h3
          className="panel-title"
        >
          Panel title 2
        </h3>
      </div>
      <div
        className="panel-body"
      >
        <div
          className="ui-panel-content"
        >
          Panel Content 2
        </div>
        <div
          className="panel-control"
        >
          <div
            className="wrap-control-group hide-indent-bottom"
          >
            <div
              className="control-group with-indent right"
            >
              <button
                className="btn btn-success btn-inverse btn-create"
              >
                <i
                  className="btn-icon fa fa-plus"
                />
                 
                <span
                  className="btn-text"
                >
                  Create
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <PanelSubAccordion /> should renders PanelSubAccordion with props correctly 3`] = `
<div
  className="col-xs-12 col-sm-6"
>
  <div
    className="ui-sub-section"
  >
    <div
      className="ui-sub-title"
    >
      Panel sub-accordion
    </div>
    <div
      className="panel panel-secondary panel-subaccordion accordion"
    >
      <div
        className="panel-heading"
        onClick={[Function]}
      >
        <h3
          className="panel-title"
        >
          Panel title 1
        </h3>
      </div>
      <div
        className="panel-body"
      >
        <div
          className="ui-panel-content"
        >
          Panel Content 1
        </div>
        <div
          className="panel-control"
        >
          <div
            className="wrap-control-group hide-indent-bottom"
          >
            <div
              className="control-group with-indent right"
            >
              <button
                className="btn btn-success btn-inverse btn-create"
              >
                <i
                  className="btn-icon fa fa-plus"
                />
                 
                <span
                  className="btn-text"
                >
                  Create
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="panel panel-secondary panel-subaccordion accordion open"
    >
      <div
        className="panel-heading"
        onClick={[Function]}
      >
        <h3
          className="panel-title"
        >
          Panel title 2
        </h3>
      </div>
      <div
        className="panel-body"
      >
        <div
          className="ui-panel-content"
        >
          Panel Content 2
        </div>
        <div
          className="panel-control"
        >
          <div
            className="wrap-control-group hide-indent-bottom"
          >
            <div
              className="control-group with-indent right"
            >
              <button
                className="btn btn-success btn-inverse btn-create"
              >
                <i
                  className="btn-icon fa fa-plus"
                />
                 
                <span
                  className="btn-text"
                >
                  Create
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Component <Panels /> should renders Panels with props correctly 1`] = `
<div
  className="ui-section"
  id="panels"
>
  <strong
    className="ui-title"
  >
    Panels
  </strong>
  <div
    className="row"
  >
    <div
      className="col-xs-12 col-sm-6"
    >
      <div
        className="ui-sub-section"
      >
        <div
          className="ui-sub-title"
        >
          Panel primary
        </div>
        <div
          className="panel panel-primary"
        >
          <div
            className="panel-heading"
          >
            <div
              className="control-group right"
            >
              <button
                className="btn btn-success btn-inverse btn-square hidden-xs hidden-sm btn-expand-panel"
              >
                <i
                  className="btn-icon fa fa-expand"
                />
                <i
                  className="btn-icon fa fa-compress"
                />
              </button>
            </div>
            <h3
              className="panel-title"
            >
              Panel title
            </h3>
          </div>
          <div
            className="panel-body"
          >
            <div
              className="ui-panel-content"
            >
              Panel Conten
            </div>
            <div
              className="panel-control"
            >
              <div
                className="wrap-control-group hide-indent-bottom"
              >
                <div
                  className="control-group with-indent right"
                >
                  <button
                    className="btn btn-success btn-inverse btn-create"
                  >
                    <i
                      className="btn-icon fa fa-plus"
                    />
                     
                    <span
                      className="btn-text"
                    >
                      Create
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="col-xs-12 col-sm-6"
    >
      <div
        className="ui-sub-section"
      >
        <div
          className="ui-sub-title"
        >
          Panel secondary
        </div>
        <div
          className="panel panel-secondary"
        >
          <div
            className="panel-heading"
          >
            <div
              className="control-group right"
            >
              <button
                className="btn btn-success btn-inverse btn-square hidden-xs hidden-sm btn-expand-panel"
              >
                <i
                  className="btn-icon fa fa-expand"
                />
                <i
                  className="btn-icon fa fa-compress"
                />
              </button>
            </div>
            <h3
              className="panel-title"
            >
              Panel title
            </h3>
          </div>
          <div
            className="panel-body"
          >
            <div
              className="ui-panel-content"
            >
              Panel Conten
            </div>
            <div
              className="panel-control"
            >
              <div
                className="wrap-control-group hide-indent-bottom"
              >
                <div
                  className="control-group with-indent right"
                >
                  <button
                    className="btn btn-success btn-inverse btn-create"
                  >
                    <i
                      className="btn-icon fa fa-plus"
                    />
                     
                    <span
                      className="btn-text"
                    >
                      Create
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-xs-12 col-sm-6"
    >
      <div
        className="ui-sub-section"
      >
        <div
          className="ui-sub-title"
        >
          Panel secondary
        </div>
        <div
          className="panel panel-secondary"
        >
          <div
            className="panel-heading"
          >
            <div
              className="control-group right"
            >
              <button
                className="btn btn-success btn-inverse btn-square hidden-xs hidden-sm btn-expand-panel"
              >
                <i
                  className="btn-icon fa fa-expand"
                />
                <i
                  className="btn-icon fa fa-compress"
                />
              </button>
            </div>
            <h3
              className="panel-title"
            >
              Panel title
            </h3>
          </div>
          <div
            className="panel-body"
          >
            <div
              className="panel-body-inner"
            >
              <div
                className="ui-panel-content"
              >
                Panel Conten
              </div>
            </div>
            <div
              className="panel-control"
            >
              <div
                className="wrap-control-group hide-indent-bottom"
              >
                <div
                  className="control-group with-indent right"
                >
                  <button
                    className="btn btn-success btn-inverse btn-create"
                  >
                    <i
                      className="btn-icon fa fa-plus"
                    />
                     
                    <span
                      className="btn-text"
                    >
                      Create
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="col-xs-12 col-sm-6"
    >
      <div
        className="ui-sub-section"
      >
        <div
          className="ui-sub-title"
        >
          Panel small
        </div>
        <div
          className="panel panel-secondary panel-small"
        >
          <div
            className="panel-heading"
          >
            <h3
              className="panel-title"
            >
              Panel title
            </h3>
          </div>
          <div
            className="panel-body"
          >
            <div
              className="ui-panel-content"
            >
              Panel Conten
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <PanelAccordion />
    <PanelSubAccordion />
  </div>
</div>
`;
